source/UniverseObjects.cpp

```diff
@@ -503,6 +503,8 @@ void UniverseObjects::LoadFile(const string &path, bool debugMode)
 			else
 				node.PrintTrace("Invalid use of keyword \"disable\" for class \"" + category + "\"");
 		}
+		else if(key == "reputation titles")
```

Comment:
We could also make them more genericly named; indexed titles or smth like that
Not needed though

Reply:
I'm not sure why that would be desirable.

Reply:
To use this kind of titles for other things, like setting tje exact level at which you get some name by plugins or smth else

Reply:
its fine as is, just noting btw these wont be modifiable by events but they shouldnt be so all good